api/index.ts

```diff
@@ -117,18 +117,120 @@ const v1Routes = (cfg: APIConfig, store: Store) => async (server: FastifyTypebox
   }
 
   // Get global list of blocked users/instances as newline delimited string
-  server.get('/blocklist', async (request, reply) => {})
+  server.get('/blocklist', {
+    schema: {
+      response: {
+        200: Type.String()
+      },
+      description: 'Get global list of blocked users/instances as newline delimited string.',
+      tags: ['ActivityPub']
```

Comment:
```suggestion
      tags: ['Moderation']
```

Since this isn't part of the activitypub spec, mind changing the tags to "moderation" so they'll show up in a seperate section?